WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -56,7 +56,7 @@ public async Task<bool> StartAsync(bool ensureRunning)
 			try
 			{
 				// Is Tor already running? Either our Tor process from previous Wasabi Wallet run or possibly user's own Tor.
-				bool isAlreadyRunning = await TorSocks5Client.IsTorRunningAsync().ConfigureAwait(false);
+				bool isAlreadyRunning = await TorSocks5ClientFactory.IsTorRunningAsync().ConfigureAwait(false);
```

Comment:
It is not. It simply attempts to connect to Tor SOCSK5 server on localhost to verify that Tor is running and it should be very fast and also it does not take a long time so that's why there is no pooling.